monai/data/torchscript_utils.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2020 - 2021 MONAI Consortium
```

Comment:
Hi @ericspod ,

@wyli and I think maybe it's better to move this utility to other folder, like: https://github.com/Project-MONAI/MONAI/pull/3138#issuecomment-995556255.

What do you think?

Thanks.

Reply:
For now we don't have enough code to justify that. There's a bit of code in network utils that may make sense to concentrate here if we wanted to have all jit-related routines in one place.

Reply:
ok, thanks for the explanation.